packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -8531,6 +8531,33 @@ describe('plugin-meetings', () => {
           });
         });
       });
+
+      describe('#brb', () => {
+        it('send brb request', () => {
```

Comment:
@SomeBody16  I’m not sure why the test is named "brb." If we are testing the "beRightBack" public method, I would expect the test name to be consistent with the API. Additionally, I would suggest changing the naming of the test case because it focuses on one specific aspect of the method, while there are multiple things involved (request, wcme api). As I mentioned in another comment, the current naming feels for me overly detailed regarding the implementation. I would suggest making it more abstract, focusing on the logic instead. Something like "should enable/disable the brb with the correct data and return a promise."
Open to discussion.